src/layout/Header/index.tsx

```diff
@@ -13,60 +14,95 @@ const Header = () => {
   const handleMouseLeave = () => {
     setIsHovered(false);
   };
+
+  const handleMouseClickOnMenuButton = () => {
+    setIsMenuShown(!isMenuShown);
+  };
+
   return (
-    <header className={styles.header}>
-      <div className={styles.content}>
-        <nav className={styles.nav}>
-          <h1 className={styles.nav__heading}>TaoTalk</h1>
-          <div className={styles.nav__divide}>|</div>
-          <ul className={styles.nav__list}>
-            <li>
-              <a title="Home" href="#" className={styles.nav__list_item}>
-                Home
-              </a>
-            </li>
-            <li>
-              <a
-                title="Textbook"
-                href="#"
-                className={[styles.nav__list_item, styles['nav__list_item--inactive']].join(' ')}
-              >
-                Textbook
-              </a>
-            </li>
-            <li>
-              <a
-                title="Statistics"
-                href="#"
-                className={[styles.nav__list_item, styles['nav__list_item--inactive']].join(' ')}
-              >
-                Statistics
-              </a>
-            </li>
-            <li
-              className="relative"
-              onMouseEnter={handleMouseEnter}
-              onMouseLeave={handleMouseLeave}
-            >
-              <a
-                title="Games"
-                href="#"
-                className={[styles.nav__list_item, styles['nav__list_item--inactive']].join(' ')}
-              >
-                Games
-                <img src="/down_arrow.svg" alt="down array"></img>
-              </a>
-              {isHovered && (
-                <div className=" absolute top-[46px] w-[215px] rounded-[14px] bg-white shadow-[0_343px_343px_0_rgba(23,64,51,0.08)] ">
-                  <DropDownMenu menuContent={['Sprint →', 'Audio-call →']} />
-                </div>
-              )}
+    <div>
+      <header className="flex h-[88px] items-center justify-center bg-tk_greyLight shadow-xl">
+        <div className="flex w-full items-center justify-between px-[8px] md:px-[32px] lg:w-[1180px] lg:px-0">
+          <nav className="flex items-center gap-[30px]">
+            <h1 className="font-Dela_Gothic_One text-[21px] font-normal tracking-[-0.04em]">
+              TaoTalk
+            </h1>
+            <div className="hidden md:flex md:gap-[30px]">
+              <span className="text-tk_greyMedium">|</span>
+              <ul className="flex list-none items-center pl-0">
```

Comment:
https://github.com/taodemy/taotalk/blob/d8f609ef26d9a3dc290c100b53baff68a5dfc43a/src/layout/Header/index.tsx#L30

I set `list-none` to eliminate the bulleting dots. I think 'list-none' is default setting in tailwind. So it doesn't need to be set.

I set 'pl-0' to avoid gap-[30px] setting in parent element adding margin left value to cause problems. But it seems unnecessary. I think `pl-0` may be the default setting.